Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php

```diff
@@ -101,7 +101,7 @@ private function fixDocBlock($content, $indent)
      * @param string $content Whitespace before Docblock
      * @param string $indent  Indentation of the documented subject
      *
-     * @return string Whitespace including correct indentation for Dockblock after this whitespace
```

Comment:
Is it not obvious? The ones that are identical to the method name, or the short description are not useful, otherwise, they probably are useful?